app/controllers/concerns/idv_session.rb

```diff
@@ -42,6 +42,30 @@ def idv_attempter
   end
 
   def vendor_validator_result
+    return timed_out_vendor_error if vendor_result_timed_out?
+
     VendorValidatorResultStorage.new.load(idv_session.async_result_id)
   end
+
+  def vendor_result_timed_out?
+    started_at = idv_session.async_result_started_at
+    return false if started_at.blank?
+
+    expiration = started_at + Figaro.env.async_job_refresh_max_wait_seconds.to_i
+    Time.zone.now.to_i >= expiration
+  end
+
+  def timed_out_vendor_error
+    Idv::VendorResult.new(
+      success: false,
+      errors: { timed_out: ['Timed out waiting for vendor response'] },
```

Comment:
I thought about that, but then I realized that it only is sent to error analytics for us to consume internally -- it's not user-facing at all, so I think it's OK to leave un-translated